src/components/sbb-teaser/sbb-teaser.e2e.ts

```diff
@@ -5,7 +5,9 @@ describe('sbb-teaser', () => {
 
   it('should forward host click to action element', async () => {
     page = await newE2EPage();
-    await page.setContent('<sbb-teaser href="link">Teaser content</sbb-teaser>');
+    await page.setContent(
```

Comment:
I'm not sure if we better remove our check for the accessibility label. we never do this elsewhere. What do you think @kyubisation? I mean this check: 

```  /**
   * Check if accessibilityLabel is provided since it is a required prop,
   * otherwise throw an error.
   */
  @Watch('accessibilityLabel')
  private _validateAccessibilityLabel(newValue: string): void {
    const isBlank = typeof newValue !== 'string' || newValue === '';
    if (isBlank) {
      throw new Error('accessibilityLabel: required');
    }
  }```

Reply:
Yes, we usually do not break things like this.
I would also suggest to remove this.